frontend/src/components/modals/pointPopup/PointPopup.tsx

```diff
@@ -0,0 +1,429 @@
+import React, { useState } from 'react';
+import { styled } from '@mui/material/styles';
+import IconButton, { IconButtonProps } from '@mui/material/IconButton';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ThumbDownAltOutlinedIcon from '@mui/icons-material/ThumbDownAltOutlined';
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+import ThumbUpOutlinedIcon from '@mui/icons-material/ThumbUpOutlined';
+import Link from '@mui/material/Link';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import SendOutlinedIcon from '@mui/icons-material/SendOutlined';
+import {
+  Card,
+  CardMedia,
+  CardContent,
+  Collapse,
+  Avatar,
+  Typography,
+  Box,
+  TextField,
+  FormControl,
+  Button,
+  List,
+  ListItem,
+  Divider,
+  ListItemText,
+  ListItemAvatar
+} from '@mui/material';
+
+// It can be reused in different components
+// When the rest of the functionality is available, this component will be assigned to a specific and each point on the map
+// Change position a photo, description, buttons, etc. you can without problems
+// Now it works on stubs
+
+interface Props {
+  active: boolean;
+  setActive: Function;
+  children: Array<any>;
```

Comment:
Why any?

Reply:
What should be there?
It's like a stub.
Data can changes many times.

Reply:
create a global interface inside types.ts file for the object you pass in
![image](https://user-images.githubusercontent.com/41287687/163573554-ef0c8651-b59c-4c09-9b5b-bd7f6a611db1.png)

Reply:
interface IPointBody {
locationName: string;
avatarMini: string;
name: string;
...
}

Reply:
create types.ts inside src folder

Reply:
This data is not final.
There may be a completely different structure and data types.

Reply:
Then interface would change